types/algoliasearch/index.d.ts

```diff
@@ -1337,81 +1339,88 @@ declare namespace algoliasearch {
      * 'none': no exact on single word query
      * 'word': exact set to 1 if the query word is found in the record
      * 'attribute': exact set to 1 if there is an attribute containing a string equals to the query
-     * https://github.com/algolia/algoliasearch-client-js#exactonsinglewordquery
+     * https://www.algolia.com/doc/api-reference/api-parameters/exactOnSingleWordQuery/
      */
-    exactOnSingleWordQuery?: string;
+    exactOnSingleWordQuery?: "attribute"|"none"|"word";
     /**
      * Specify the list of approximation that should be considered as an exact match in the ranking formula
      * default: ['ignorePlurals', 'singleWordSynonym']
      * 'ignorePlurals': alternative words added by the ignorePlurals feature
      * 'singleWordSynonym': single-word synonym (For example "NY" = "NYC")
      * 'multiWordsSynonym': multiple-words synonym
-     * https://github.com/algolia/algoliasearch-client-js#alternativesasexact
+     * https://www.algolia.com/doc/api-reference/api-parameters/alternativesAsExact/
      */
-    alternativesAsExact?: any;
+    alternativesAsExact?: string[];
```

Comment:
This seems more correct 🙂 

```suggestion
    alternativesAsExact?: Array<"ignorePlurals " | "singleWordSynonym" | "multiWordsSynonym">;
```

Reply:
Very good catch! I must've missed it. I've applied your suggested changes without the extra space at the end on `ignorePlurals` :)